news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,212 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>
+
+---
+## New Features on osu!lazer!
+The new client has never been that **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
+
+Here's the [patch note](https://osu.ppy.sh/home/changelog), and the showcase of **the latest mod**! 
+
+To start with this, we want to showcase a pretty chaotic but fancy Repel mod (finally a worthy opponent for badeu?)! 
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+---
+That said, you're maybe thinking that this Lazer section isn't long enough. Right ? Here's a section about Lazer's UI design, wrote by [0x84f](https://osu.ppy.sh/users/7944724).
+
+
+
+## Summary of this Article
+### ![](/wiki/shared/mode/osu.png) osu!
+- The Roundtable has ended on insane matches, especially the Utami vs mrekk one! 
+  
+  [➥ Find the article here!](#if-you-ask-us-what-is-the-big-event-of-july-itd-be-the-roundtable)
+
+
+- For the month's top score, we got [[Karcher]](https://osu.ppy.sh/users/11443437) achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991)!
+  
+  [➥ Find the article here!](#as-if-the-impossible-could-not-be-done-only-once-but-twice)
+  
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+- The osu!mania World Cup 2022 is the road to the greatness for the Mania top players! They will brawl again in this great arena! Who's gonna take the crown ?
+  
+  [➥ Find the article here!](#in-august-the-osumania-top-players-will-brawl-in-the-osumania-world-cup-2022)
+
+
+- The scores now!
+  Let's start with a 7K score : Onta_Bekasi only lost 0,02 accuracy on the surprisingly difficult *"My Man"*! A flawless performance!
+
+  [➥ Find the article here!](#the-mania-k-pop-community-is-going-to-be-absolutely-ecstatic-at-this-next-showcase)
+
+- Now, an amazing 4K score! If you thought *My Man* had LNs, wait 'till you watch just over 7 minutes of almost non-stop LN action along with being bridled by intense single-note action and 
+BPM changes throughout the map. [GS] Myuka has done a rampage on the Camellia’s - MI11I0N PP]!
+
+  [➥ Find the article here!](#wow-just-completely-shocked-at-this-performance-by-gsmyuka-on-camellias---mi11i0n-pp)
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+- The Catch World Cup 2022 has ended on pretty amazing performances ! Is South Korea dominating two modes at the same time ?
```

Comment:
- The Catch World Cup 2022 has ended on pretty amazing performances ! Is South Korea dominating two modes at the same time?

Reply:
Fixed.